### PR TITLE
[PR] Implement post-process of 3d pose estimation (SimpleBaseline)

### PR DESCRIPTION
**Pull request by tucan9389**
| demo |
| --- |
| simplebaseline-3dpose-demo-stillimage |

## Related Issue

- https://github.com/tucan9389/PoseEstimation-TFLiteSwift/issues/28

## PR Points

- Implement the pre-process
  - Image normalization using in PyTorch usually
- Implement the post-process
  - Define the keypoints each index
  - Implement softmax
  - Implement sum function that could get dim as augment like numpy or pytorch
  - Optimize sum and mul with Accelerate framework
- Log the 3d points
- Draw the 3d points in the app screen
- Support real-time (later)
- Implement additional post-processing for removing some noises (later)

### CODE REVIEW

**Comment by tucan9389** [comment]
Currently, the output of the post-processing is incorrect now. I'm asking the post-processing method to the model engineer now.

**Comment by tucan9389** [comment]
I implemented the 3d pose estimation post-processing now. I fixed the original argmax to softmax argmax.
